samples/04_gis_analysts_data_scientists/forecasting_monthly_rainfall_in_California_using_deeplearning_timeseries_model_from_arcgis_learn.ipynb

```diff
@@ -0,0 +1,2227 @@
+{
```

Comment:
<p>If this cell is extra remove it and if this cell will be used in some specific condition please add one or two line explaining how and when this line can be used.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>the use of preprocessors is now explained with a brief, in case of scaler functions</p>